elide-integration-tests/src/test/java/com/yahoo/elide/async/integration/tests/TableExportIT.java

```diff
@@ -0,0 +1,1250 @@
+                         + "{ url httpStatus recordCount } } } } }\","
+                         + "\"variables\":null}")
+                 .post("/graphQL")
+                 .asString();
+
+        expectedResponse = "{\"data\":{\"tableExport\":{\"edges\":[{\"node\":{\"id\":\"edc4a871-dff2-4054-804e-d80075cf829e\",\"queryType\":\"GRAPHQL_V1_0\",\"status\":\"COMPLETE\","
+                 + "\"result\":{\"url\":\"http://localhost:" + port + "/export/edc4a871-dff2-4054-804e-d80075cf829e\","
+                 + "\"httpStatus\":200,\"recordCount\":3}}}]}}}";
+
+        assertEquals(expectedResponse, responseGraphQL);
+        assertEquals("[\n"
+                + "{\"title\":\"Ender's Game\"}\n"
+                + ",{\"title\":\"Song of Ice and Fire\"}\n"
+                + ",{\"title\":\"For Whom the Bell Tolls\"}\n"
+                + "]\n", getStoredFileContents(port, "edc4a871-dff2-4054-804e-d80075cf829e"));
+    }
+
+    /**
+     * Test for QueryStatus Set to PROCESSING instead of Queued.
+     */
+    @Test
+    public void graphQLTestCreateFailOnQueryStatus() {
+
+        TableExport queryObj = new TableExport();
+        queryObj.setId("edc4a871-dff2-4054-804e-d80075cf839e");
+        queryObj.setAsyncAfterSeconds(0);
+        queryObj.setQueryType("GRAPHQL_V1_0");
+        queryObj.setStatus("PROCESSING");
+        queryObj.setResultType("CSV");
+        queryObj.setQuery("{\"query\":\"{ book { edges { node { id title } } } }\",\"variables\":null}");
+        String graphQLRequest = document(
+                mutation(
+                        selection(
+                                field(
+                                        "tableExport",
+                                        arguments(
+                                                argument("op", "UPSERT"),
+                                                argument("data", queryObj, UNQUOTED_VALUE)
+                                        ),
+                                        selections(
+                                                field("id"),
+                                                field("query"),
+                                                field("queryType"),
+                                                field("status")
+                                        )
+                                )
+                        )
+                )
+        ).toQuery();
+
+        JsonNode graphQLJsonNode = toJsonNode(graphQLRequest, null);
+        ValidatableResponse response = given()
+                .contentType(MediaType.APPLICATION_JSON)
+                .accept(MediaType.APPLICATION_JSON)
+                .body(graphQLJsonNode)
+                .post("/graphQL")
+                .then()
+                .statusCode(org.apache.http.HttpStatus.SC_OK);
+
+        String output = response.extract().body().asString();
+        assertEquals(true, output.contains("errors"));
+        assertEquals(true, output.contains("CreatePermission Denied"));
```

Comment:
Why does this fail with create permission?

Reply:
It fails the security check `@SecurityCheck(AsyncAPIStatusQueuedValue.VALUE_IS_QUEUED)`